apps/marketplace/components/BuyerDashboard/BuyerDashboardMyBriefs.js

```diff
@@ -0,0 +1,148 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import ClosedDate from 'shared/ClosedDate'
+import LoadingIndicatorFullPage from 'shared/LoadingIndicatorFullPage/LoadingIndicatorFullPage'
+import { loadBuyerDashboard } from 'marketplace/actions/dashboardActions'
+import { BUYER_DASHBOARD_MYBRIEFS_SUCCESS } from 'marketplace/constants/constants'
+import { statusConvert } from 'marketplace/components/helpers'
+import { rootPath } from 'marketplace/routes'
+import BuyerDashboardHelp from './BuyerDashboardHelp'
+import styles from './BuyerDashboard.scss'
+
+export class BuyerDashboardMyBriefs extends Component {
+  componentDidMount() {
+    this.props.loadData(BUYER_DASHBOARD_MYBRIEFS_SUCCESS, '/dashboard/my/briefs')
```

Comment:
not sure about the component knowing about this constant - should really create actions for each case